Unreal/CarlaUnreal/Plugins/Carla/Source/Carla/Actor/ActorBlueprintFunctionLibrary.cpp

```diff
@@ -1523,14 +1523,14 @@ FColor UActorBlueprintFunctionLibrary::RetrieveActorAttributeToColor(
 // editor crashing while people are testing new actor definitions.
 #if WITH_EDITOR
 #  define CARLA_ABFL_CHECK_ACTOR(ActorPtr)                    \
-  if ((ActorPtr == nullptr) || ActorPtr->IsPendingKill())     \
+  if ((ActorPtr == nullptr) || IsValid(ActorPtr))     \
   {                                                           \
     UE_LOG(LogCarla, Error, TEXT("Cannot set empty actor!")); \
     return;                                                   \
   }
 #else
 #  define CARLA_ABFL_CHECK_ACTOR(ActorPtr) \
-  check((ActorPtr != nullptr) && !ActorPtr->IsPendingKill());
+  check((ActorPtr != nullptr) && IsValid(ActorPtr) );
```

Comment:
```suggestion
  check((ActorPtr != nullptr) && !IsValid(ActorPtr));
```
I think you wanted to look for non valid Actors to throw an error right?

Reply:
No. the check will trigger if one of conditions are false. We want to check if the ActorPtr is not null ptr and if it is valid